front-end/src/app/shared/models/transaction-type.model.ts

```diff
@@ -75,7 +76,7 @@ export abstract class TransactionType {
   formTitle?: string; // Title of form within accordion section
   footer?: string; // Text at the end of form
   contactTitle?: string; // Title for primary contact
-  contactLookupLabel?: string; //Label above contact lookup
+  contactLookupLabel = "LOOKUP"; // Set label for all lookups
```

Comment:
Recommend removing this property from transaction type models if its not variable any longer